client/src/hooks/queries/useGetDruggableSources.ts

```diff
@@ -13,6 +13,8 @@ query sources($sourceType: SourceTypeFilter) {
     }
     nodes {
       sourceDbName
+      sourceDbVersion
```

Comment:
I think you're right that these queries are identical -- we should make a separate issue to clean it up